plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/controls/ScriptSelectorPanel.java

```diff
@@ -333,6 +371,14 @@ public void keyPressed(KeyEvent e) {
         return composite;
     }
 
+    private void setDialogInput(Button projectCheckbox) {
```

Comment:
What if someone wants to call this method without having an instance of some kind of `Button` to pass in?

Please pass `boolean` instead and provide a meaningful name for this parameter.